cmd/cmd_bucket.go

```diff
@@ -154,7 +148,7 @@ func createBucket(ctx *cli.Context) error {
 		opts.ChargedQuota = chargedQuota
 	}
 
-	txnHash, err := client.CreateBucket(c, bucketName, primarySpAddr, opts)
+	txnHash, err := client.CreateBucket(c, bucketName, primarySpAddrStr, opts)
```

Comment:
even there is txHash return, it is not guaranteed that it is succeed, would suggest check the status code == 0 or not to confirm

Reply:
change to broad cast mode to BroadcastMode_BROADCAST_MODE_BLOCK